artman/cli/main2.py

```diff
@@ -136,13 +136,15 @@ def parse_args(*args):
         help='[Optional] Directory to store output generated by artman. '
         'Default to `./artman-genfiles`', )
     parser.add_argument(
-        '--input-dir',
+        '--root-dir',
         type=str,
         default='',
         help='[Optional] Directory with all input that is needed by artman, '
-             'which include but not limited to API protos, service config '
-             'yaml, and GAPIC config yaml. Default to the settings from user '
-             'configuration.',
+        'which include but not limited to API protos, service config yaml '
```

Comment:
nit: "include but not limited to" -> "includes but is not limited to"

Reply:
Updated.